### ISSUE TITLE
Editor: cannot add a poll when creating a post

### ISSUE THREAD

**Issue by designsimply** [Automattic/wp-calypso#1791]
Raised by @justinshreve 

If we can do this in a similar way to the media explorer overrides, we should do it. If it requires a lot of investment I don't think its worth it.

WP Admin screenshots for reference:

![screen shot 2015-06-03 at wed jun 3 9 07 00 pm](https://cloud.githubusercontent.com/assets/1119271/7974910/d164c790-0a35-11e5-86bf-091969994de9.png)

![screen shot 2015-06-03 at wed jun 3 9 04 52 pm](https://cloud.githubusercontent.com/assets/1119271/7974915/db79adea-0a35-11e5-9950-aa562b12dbb6.png)

![screen shot 2015-06-03 at wed jun 3 9 05 16 pm](https://cloud.githubusercontent.com/assets/1119271/7974917/e170b6a8-0a35-11e5-806a-689262999483.png)

<sup>Seen at https://madefortesting478.wordpress.com/wp-admin/post-new.php using Firefox 38.0.1 on Mac OSX 10.10.3 logged in as user358</sup>

#8556 calypso-gaps

**Comment by drw158** [comment]
I think developing a poll creation tool in Calypso is not within scope and very low priority. Somehow adding the tools via an iframe(?) as shown above will create a bad experience.

I propose that we:

- at least get Polldaddy embed previews working #10866
- #1729 will enable a UI for copy/pasting embed links, including Polldaddy
- update Polldaddy's instructions:

![screen shot 2017-01-23 at 3 17 49 pm](https://cloud.githubusercontent.com/assets/618551/22223086/25499cf6-e17f-11e6-9293-2d749133c3ea.png)

Basically, just transfer all poll creation to Polldaddy itself instead of doing it within Calypso.

Alternatively, we could add an additional insert item alongside Add Contact Form and Add Media, that just shows a simple UI and explanatory text for copy/pasting the embed link.

**Comment by lancewillett** [comment]
Closing in favor of #10866 and #1729.
